packages/preview/orange-book/0.6.1/README.md

```diff
@@ -0,0 +1,72 @@
+# orange-book
+A book template inspired by The Legrand Orange Book of Mathias Legrand and Vel https://www.latextemplates.com/template/legrand-orange-book.
+
+## Usage
+You can use this template in the Typst web app by clicking "Start from template"
+on the dashboard and searching for `orange-book`.
+
+Alternatively, you can use the CLI to kick this project off using the command
+```
+typst init @preview/orange-book
+```
+
+Typst will create a new directory with all the files needed to get you started.
+
+## Configuration
+This template exports the `book` function with the following named arguments:
+
+- `title`: The book's title as content.
+- `subtitle`: The book's subtitle as content.
+- `author`: Content or an array of content to specify the author.
+- `paper-size`: Defaults to `a4`. Specify a [paper size
+  string](https://typst.app/docs/reference/layout/page/#parameters-paper) to
+  change the page format.
+- `copyright`: Details about the copyright or
+  `none`.
+- `lowercase-references`: True to have references in lowercase (Eg. table 1.1)
+
+The function also accepts a single, positional argument for the body of the
+book.
+
+The template will initialize your package with a sample call to the `book`
+function in a show rule. If you, however, want to change an existing project to
+use this template, you can add a show rule like this at the top of your file:
+
+```typ
+#import "@preview/orange-book:0.6.1": book
+
+#show: book.with(
```

Comment:
It was probably the case with previous version too, but I just noticed that the arguments below still use camelCase and not kebab-case. Can you please fix that?

Reply:
Done, thanks for notice it!